### DISCUSSION TITLE
How to run the simple simulink

### DISCUSSION THREAD

**Discussion by mjmirzaei** [NatLabRockies/ROSCO#206] [Announcements]
I want to run Simulink for the first time, and I see some errors as follows.

Error in S-function '[OpenLoop/FAST Nonlinear Wind Turbine/S-Function](matlab:open_and_hilite_hyperlink ('OpenLoop/FAST Nonlinear Wind Turbine/S-Function','error'))': S-Function 'FAST_SFunc' does not exist

Is there any step-by-step guide for running Simulink files?

**Comment by dzalkind** [comment]
If you're on a Windows machine, you can get the FAST_SFunc and library required from here: https://github.com/OpenFAST/openfast/releases/tag/v3.3.0 

Unfortunately, we do not have a guide at this time.  Perhaps the OpenFAST team does:

https://github.com/OpenFAST/openfast/tree/main/glue-codes/simulink
